packages/client-react-streaming/src/DataTransportAbstraction/printMinified.tsx

```diff
@@ -0,0 +1,15 @@
+import type { DocumentNode } from "@apollo/client/index.js";
+import { print } from "@apollo/client/utilities/index.js";
+
+export function printMinified(query: DocumentNode): string {
+  return (
+    print(query)
+      // replace multi-spaces with single space
+      .replace(/\s{2,}/g, " ")
+      // remove spaces that are preceeded by braces
+      .replace(/(?<=[{}])\s+/g, "")
+      // remove spaces that are preceeding braces
+      .replace(/\s+(?=[{}])/g, "")
+      .trim()
```

Comment:
It looks like `graphql` has a [`stripIgnoredCharacters`](https://github.com/graphql/graphql-js/pull/1802) utility that handles this. Can we use that function to handle stripping the whitespace? It looks like that utility is token-aware so it should be a bit more accurate.

Reply:
Hmm, that pulls in about 10kb more bundle size - but since we're in SSR world here, that should be okay.
We might at some point get to the point that we pull in too much for edge functions, though (they have a limit of 1MB I believe?), so I'll make a note that that's a great place for a future size optimization.

Reply:
Ah forget about it, most of that is included in `gql` already, so it's likely fine anyways.